k8s/plugin/src/resources/errors.rs

```diff
@@ -0,0 +1,58 @@
+use snafu::Snafu;
+// use thiserror::Error;
```

Comment:
hmm I see thiserror which actually probably pairs up better with anyhow, any reason to use snafu instead?

Reply:
removed the errors file and did a simple match on errors